services/movie.py

```diff
@@ -0,0 +1,45 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids
+        ).filter(actors__id__in=actors_ids)
```

Comment:
The current filtering logic with two separate `filter` calls can be optimized by using a single `filter` call with both conditions combined using `Q` objects. This would be more efficient and concise.